### PR TITLE
Add ADR about moving content from messages file to views

### PR DESCRIPTION
**Pull request by suzannehamilton**

### CODE REVIEW

File: architecture-decision-records/0019-remove-frontend-messages-file.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+[built into the Play framework][play-i18n]. We added it when we first started
+building TDR in late 2019 because we thought we might have to add multiple
+translations to the frontend.
+
+It now seems quite unlikely that we'll need to add translations soon, and we've
+encountered a few downsides to the messages file:
+
+* It makes it harder to update the HTML templates, because content is referred
+  to indirectly.
+* This is particularly difficult in content-heavy pages. See this [draft commit
```
Comment (TomJKing) [inline architecture-decision-records/0019-remove-frontend-messages-file.md:24]: This second point is just an example of the difficulties caused by the first point. I would have this as a sub-point of the first point.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a few minor comments.

File: architecture-decision-records/0019-remove-frontend-messages-file.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  of the help guide][help-guide], for example, where it's very hard to follow
+  the content on the page.
+* It makes it harder to see whether the assertions are correct in the route
```
[the remark this answers is not in the document]
Comment (suzannehamilton) [reply architecture-decision-records/0019-remove-frontend-messages-file.md]: Fixed, thanks.

File: architecture-decision-records/0019-remove-frontend-messages-file.md (hunk 1 above)
Comment (suzannehamilton) [reply architecture-decision-records/0019-remove-frontend-messages-file.md]: Done.

File: architecture-decision-records/0019-remove-frontend-messages-file.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Decision
+
+The dev team had a discussion on 2021-05-20, and there was a general consensus
+that we should move the content out of the messages file and into the view
+templates themselves.
+
+If we ever do have to internationalise TDR (which isn't part of the upcoming
+roadmap), we can revisit this decision. It may be that only some pages need to
+be translated. And if we do need to translate content-heavy pages like the help
```
[the remark this answers is not in the document]
Comment (suzannehamilton) [reply architecture-decision-records/0019-remove-frontend-messages-file.md:48]: Are you objecting to starting a sentence with "And", or do you mean something else?

[the remark this answers is not in the document]
Comment (TomJKing) [reply architecture-decision-records/0019-remove-frontend-messages-file.md:48]: That, and also starting a new paragraph.

[the remark this answers is not in the document]
Comment (suzannehamilton) [reply architecture-decision-records/0019-remove-frontend-messages-file.md]: I've made that change, thanks.

I think it's generally OK to start a sentence with "And", though. 🙂 https://english.stackexchange.com/questions/459554/starting-a-sentence-with-and

**Review by TomJKing — APPROVED** [review]
